docs/anatomy/anatomy-environment.md

```diff
@@ -25,6 +25,43 @@ This feature is only available for **Pay-As-You-Go** users. Free users are limit
 5. Create a name for your environment.
 6. Start adding services to your environment
 
+## Edit Environment
+
+1. Login to your account
+2. Click on the **Cogwheel** icon displayed at the top-right corner.
+
+![Edit Environment](/img/anatomy/edit-environment.png)
+
+3. Edit your **Environment name** in the slide panel displayed.
+4. Click on the **Save** button
+
+On clicking `Save` button, your edited `Environment Name` will be saved.
+
+![Save Environment](/img/anatomy/edit-save.png)
+
+:::note
+1. The **Region** field is locked and cannot be edited. 
+2. The **Environment ID** is a unique ID generated for each environment created and can be copied.
```

Comment:
action -  is a unique id for your environment and may be requested by the support team when aiding you with an issue.

Reply:
Done